src/main/scala/org/scalajs/tools/tsimporter/sc/Definitions.scala

```diff
@@ -8,8 +8,10 @@ package org.scalajs.tools.tsimporter.sc
 import scala.language.implicitConversions
 
 import scala.collection.mutable._
+import scala.collection.immutable
```

Comment:
This is not necessary. `immutable.Set` is the same as `Set`, so use the latter instead.